docs/source/data_modeling.rst

```diff
@@ -63,6 +63,69 @@ Apply view
 ^^^^^^^^^^^^^^^^^
 .. automethod:: cognite.client._api.data_modeling.views.ViewsAPI.apply
 
+Example on creating a view with a direct relation and edge relation
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+.. code:: python
```

Comment:
Great effort in creating an example, lets just add it as an example in the docstring of the `view.apply` method. https://github.com/cognitedata/cognite-sdk-python/blob/master/cognite/client/_api/data_modeling/views.py#L241C61-L241C61